PURL-TYPES.rst

```diff
@@ -221,6 +207,34 @@ nuget
 
       pkg:nuget/EnterpriseLibrary.Common@6.0.1304
 
+oci-artifact
+------------
+``oci-artifact`` for all artifacts stored in OCI Distribution-spec conformant
+registries including container images built by docker and others:
+
+- There is no canonical package repository for OCI artifacts. Therefore
+  ``oci-artifact`` pURLs must be registry agnostic.
+- OCI pURLs do not require a ``namespace``. The ``namespace`` is specific
+  to the physical location of the package.
+- The ``name`` is not case sensitive and must be lowercased. The name is the
+  last fragment of the repository name. For example if the repository
+  name is ``library/debian`` then the ``name`` is ``debian``.
+- The ``version`` is the ``@sha256:digest`` of the artifact and is strongly
+  suggested for oci artifacts.
+- Optional qualifiers may include:
+
+  - ``arch``: key for a package architecture, when relevant
+  - ``repository``: A repository URL where the artifact may be found, but not intended as the only location
```

Comment:
Could we use `repository_url` to be consistent with other types?

Reply:
Works for me

Reply:
Is this a URL or a repository name, and if the latter, does it include the registry? A repository name would be `example.com:5000/group/app`, where the URL would need the schema in front (e.g. `https://`) which isn't normally seen in container image references.

Reply:
If you take a look in the examples, we are proposing something like `docker.io/library/debian` for this value.

Reply:
So instead of "repository URL", I'd recommend calling it the "OCI repository". Calling it a URL implies to me I could `curl docker.io/library/debian`, which isn't a feature of OCI.